modules/standard/FileSystem.chpl

```diff
@@ -1071,14 +1015,6 @@ proc isMount(name: string): bool throws {
   return ret != 0;
 }
 
-@deprecated(notes="listdir is deprecated, please use listDir instead")
```

Comment:
Is there also a standalone iterator version

Reply:
This is the only listdir I see in `FileSystem.chpl`, and the test doesn't have a `forall` case indicating that it does not.